.github/workflows/npmpublish.yml

```diff
@@ -0,0 +1,21 @@
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  publish-npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      - run: yarn
+      - run: yarn build
+      - run: yarn test
+      - run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.npm_token}}
```

Comment:
Where does the `npm_token` come from?

Reply:
![image](https://user-images.githubusercontent.com/8686039/67391298-b150a100-f59e-11e9-8187-eb0133a99823.png)

Reply:
Coquinou